client/me/help/help-contact/index.jsx

```diff
@@ -190,6 +194,7 @@ const HelpContact = React.createClass( {
 			this.props.isDirectlyUninitialized
 		) {
 			this.props.initializeDirectly();
+			analytics.tracks.recordEvent( 'calypso_help_contact_directly_initialize' );
```

Comment:
If we want to just track that the help page triggered the directly initialization then we can just dispatch the `recordEvent` action here